src/moonlink/src/storage/mooncake_table.rs

```diff
@@ -332,15 +333,83 @@ impl IcebergPersistedRecords {
     pub fn get_file_indices_to_reflect_persistence(&self) -> (HashSet<FileId>, Vec<FileIndex>) {
         let mut persisted_file_indices = vec![];
         persisted_file_indices.extend(self.import_result.new_file_indices.iter().cloned());
+        persisted_file_indices.extend(
```

Comment:
This was a bug: all persisted file indices should be reflected to mooncake snapshot.